src/landscape/components/LandscapeForm/BoundaryStepNewBoundariesFile.test.js

```diff
@@ -23,6 +23,10 @@ import * as terrasoApi from 'terraso-client-shared/terrasoApi/api';
 import mapboxgl from 'gis/mapbox';
 import LandscapeNew from 'landscape/components/LandscapeForm/New';
 
+jest.mock('custom-hooks', () => ({
+  useIsMounted: () => false,
+}));
```

Comment:
@nobe4 Shouldn't the mock be?
```suggestion
jest.mock('custom-hooks', () => ({
  useIsMounted: () => { current: false },
}));
```

Reply:
Good catch 👍